Add checkbox to denote backporting required on issue templates (#3669)
k3s-io/k3s: .github/ISSUE_TEMPLATE/bug_report.md

**Additional context / logs:**
<!-- Add any other context and/or logs about the problem here. -->

**Backporting**
<!--

> Does this PR need to be backported to older releases?  If so, please check this box to ensure no necessary backports are missed."

Reviewer: ```suggestion
Does this fix need to be backported to older releases?  If so, please check this box and list the branches it should be backported to.
```
